src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_trace.py

```diff
@@ -302,6 +302,7 @@ def test_spans_persist_and_line_run_gets(self, pf: PFClient) -> None:
         }
         assert terminated_line_run._to_rest_object() == expected_terminated_line_run_dict
 
+    @pytest.mark.skipif(pytest.is_replay, reason="BUG 3178603, recording instable")
```

Comment:
maybe also revert changes inside this file

Reply:
OK